cmd/root.go

```diff
@@ -216,6 +216,23 @@ var convertCmd = &cobra.Command{
 	},
 }
 
+var updateCmd = &cobra.Command{
+	Use:     "update",
+	Short:   "update <course file>",
+	Long:    "Only install/upgrade a release if there are changes.",
+	PreRunE: validateArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := reckoner.NewClient(courseFile, version, runAll, onlyRun, true, dryRun, createNamespaces, courseSchema)
+		if err != nil {
+			klog.Fatal(err)
+		}
+		err = client.Update()
+		if err != nil {
+			klog.Fatal(err)
```

Comment:
Did we want to stick with the os.Exit pattern and red error logs rather than fatal?

Reply:
Yes, we will. I was going to do a logging PR once some main functionality was done. I probably should have just started at that state for this command. I'll fix it.